### ISSUE TITLE
Extend peer dependencies to Angular 8

### ISSUE THREAD

**Issue by alvaro-octal** [isaacplmann/ngx-tour#179]
**Issue description**

running `npm install` throws the following warning:

```
npm WARN ngx-tour-core@4.0.1 requires a peer of @angular/common@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-core@4.0.1 requires a peer of @angular/core@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-core@4.0.1 requires a peer of @angular/router@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-ngx-bootstrap@4.1.0 requires a peer of @angular/common@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-ngx-bootstrap@4.1.0 requires a peer of @angular/core@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-tour-ngx-bootstrap@4.1.0 requires a peer of ngx-bootstrap@^3.1.0 but none is installed. You must install peer dependencies yourself.
```
**Steps to reproduce and a minimal demo of the problem**

`npm install` with Angular 8

**Current behavior**

`npm install` shows a couple of warning due to peer dependencies.

**Expected/desired behavior**

No warnings should be shown

**Comment by magictiti** [comment]
In a fork: https://github.com/BobBDE/ngx-tour/commit/bc9e5589050818ede2a111f1c04396d266be3ed1

**Comment by isaacplmann** [comment]
Fixed in version 4.2.0.  Please reopen if this is still a problem.
